src/components/common.css

```diff
@@ -17,4 +17,9 @@
 .sl-panel {
     height: 100%;
     flex: 1;
+}
+
+/** PDF Slick component */
+.pdfSlick {
```

Comment:
CSS for specific page can be loaded in place:
```typescript
import cssClassName from './pdfSlick.css'
```
Then, `cssClassName` would be a string containing the correct CSS class name.

Reply:
Interesting. I do want to switch to SCSS asap; writing plain CSS is nightmarish

Reply:
That's also good.